app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -754,7 +754,7 @@ private void launchNextActivityOnAppInstall() {
                 Intent i = new Intent(getApplicationContext(), DispatchActivity.class);
                 startActivity(i);
             }
-        } else if (getCallingActivity().getPackageName().startsWith(BuildConfig.APPLICATION_ID)){
+        } else if (getCallingActivity().getPackageName()!= null && getCallingActivity().getPackageName().startsWith(BuildConfig.APPLICATION_ID)){

```

Comment:
null check should be on `getCallingActivity` instead.

Reply:
🤦my bad, I misread the exception message.